packages/ui-kit/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  extends: ['@commercetools-frontend/eslint-config-mc-app'],
```

Comment:
I would avoid having nested eslint configs. You can use `overrides` in the main `.eslintrc.js` file to target specific things.